getting_started/step_by_step/ui_code_a_life_bar.rst

```diff
@@ -461,7 +465,7 @@ bar as well when the character died. We will reuse the same ``Tween``
 node as it manages multiple animations in parallel for us.
 
 First, the ``GUI`` needs to connect to the ``Player``'s ``died`` signal
-to know when it just died. Press :kbd:``F1`` to jump back to the 2D
+to know when it just died. Press ``F1`` to jump back to the 2D
```

Comment:
`:kbd:` is [valid reST](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-kbd), but it should likely be:
<pre>
:kbd:`F1`
</pre>

Reply:
Ahh thank you. I wasn't familiar with the syntax, so I assumed it was an error. I've made the correction.